sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+friendly_name: Agg Active Users
+description: |-
+  Active users aggregated by product and business dimensions.
```

Comment:
would it be worth including in the description what do we mean by product and business dimensions?